ExotischNederland/Models/Model.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ExotischNederland.Models
+{
+    internal class Model
```

Comment:
suggestion(non-blocking): I like this idea, but it will impact the class diagram, so we should document it accurately. We should start making a implementation class diagram with a 1 to 1 match with the codebase.